### PR TITLE
Report index size.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: executor/src/witgen/machines/fixed_lookup_machine.rs [hunk 1]
```diff
@@ -120,14 +121,21 @@ impl IndexedColumns {
             )
             .0;
 
+        log::trace!(
+            "Done creating index. Size (as flat list): entries * (num_inputs * input_size + row_pointer_size) = {} * ({} * {} bytes + {} bytes) = {} bytes",
+            index.len(),
+            input_column_values.len(),
+            mem::size_of::<FieldElement>(),
```
Comment (chriseth) [inline executor/src/witgen/machines/fixed_lookup_machine.rs:128]: Is there a way to make this directly depend on the type of `index` (i.e. it changes when the type changes)?

File: executor/src/witgen/machines/fixed_lookup_machine.rs [hunk 2]
```diff
@@ -120,14 +121,21 @@ impl IndexedColumns {
...
+            mem::size_of::<Option<DegreeType>>(),
+            index.len() * mem::size_of::<Option<DegreeType>>()
```
Comment (chriseth) [inline executor/src/witgen/machines/fixed_lookup_machine.rs:130]: Oh and this is even wrong.

File: executor/src/witgen/machines/fixed_lookup_machine.rs (hunk 1 above)
Comment (Schaeff) [reply executor/src/witgen/machines/fixed_lookup_machine.rs:128]: Make what depend?

Comment (chriseth) [reply executor/src/witgen/machines/fixed_lookup_machine.rs:128]: I would like to use something like `mem::size_of::<typeof::<index>::Value>()` here

**Pull request closed without merging**